scripts/run_update_branch.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Copyright 2019 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x
+set -e
+
+WD=$(dirname "$0")
+WD=$(cd "$WD"; pwd)
+ROOT=$(dirname "$WD")
+
+function update_branch() {
+    local FROM="${1}"
+    local TO="${2}"
+
+    if [ "${FROM}" != "${TO}" ]; then
+        echo "Updating version for branch ${TO}..."
+        # Update version string in docs.
+        sed -i "s|blob/${FROM}|blob/${TO}|g" "${ROOT}"/ARCHITECTURE.md
+        sed -i "s|blob/${FROM}|blob/${TO}|g" "${ROOT}"/README.md
+        # Update tag for buildin profiles.
+        find "${ROOT}"/data/profiles -type f -exec sed -i "s/tag: ${FROM}-latest-daily/tag: ${TO}-latest-daily/g" {} \;
+        # Update tag for testdata.
+        find "${ROOT}"/cmd/mesh/testdata -type f -exec sed -i "s/tag: ${FROM}-latest-daily/tag: ${TO}-latest-daily/g" {} \;
+        find "${ROOT}"/pkg/values/testdata -type f -exec sed -i "s/tag: ${FROM}-latest-daily/tag: ${TO}-latest-daily/g" {} \;
```

Comment:
@morvencao I realize this is merged but I think it's missing updating the strings inside version dir: https://github.com/istio/operator/tree/master/version - would you mind adding to this script if appropriate?

Reply:
Sure